targets/TARGET_ublox/TARGET_R5030/device/TOOLCHAIN_ARM_STD/r5030.sct

```diff
@@ -22,16 +22,16 @@
  *************************************************************
  */
 
-LR_IROM1 0x00000000 0x00040000  {    ; load region size_region
+; R5030_APPS_PROC: 128 KB FLASH (0x200000) + 1024 KB SRAM (0x100000)
```

Comment:
Good to have this comment: I guess the FLASH portion is not mapped in yet (as I can't see anything of size `0x200000` below)?

Reply:
Yes. Its a mistake in the comment. Its 0x20000

Reply:
@RobMeades changes have been made in another commit. Kindly review when possible. Thanks
(https://github.com/fahim-ublox/mbed-os/pull/3/commits/202a7c5edab21030cc5f68baa7bbd35ee1c3011f)

Reply:
Alright, kindly approve if there is no other discrepancy. Thanks